### DISCUSSION TITLE
Wino unlimited and focus

### DISCUSSION THREAD

**Discussion by kizzed** [bkaankose/Wino-Mail#359] [General]
I was wondering if I am maybe just missing how to reinstate my unlimited wino? Also is there a way to get rid of the focus tab? Thanks in advance!

**Comment by bkaankose** [comment]
Your store purchases are tied to your Microsoft account that you downloaded your account in Store app. 

Regarding focus tab, go to Manage Accounts, click your Outlook account and disable this option:

![image](https://github.com/user-attachments/assets/4640fde0-1fe6-43ff-be52-7b0f0419c66e)

**Reply by kizzed** [reply]
Edit... Figured out the focus. Cant change it after you link inboxes

**Reply by kizzed** [reply]
Well, I am on my Microsoft account that does everything...  I will look into it more. Thank you for the fast response!!

**Comment by kizzed** [comment]
I just decided to purchase it again. If it is on my ex-wifes account then that is just another thing she got in the divorce! lol

**Reply by bkaankose** [reply]
I can give you a promo code to cover that at least if you message me in Discord :)

**Reply by kizzed** [reply]
NOOOOOO! I don;t mind paying for it, you put thhe time in after all!
